packages/verification-vue-component/src/style/components/verification/_file_selection.scss

```diff
@@ -0,0 +1,104 @@
+.verification-dropbox, .verification-file-selector {
+    border-width: $verification-dropbox-border-width;
+    border-style: $verification-dropbox-border-style;
+    border-color: $verification-blue;
+
+    .input-file {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        cursor: pointer;
+        opacity: 0;
+    }
+
+    .content {
+        display: flex;
+        max-width: 430px;
+        margin: 0 auto;
+        padding: 43px;
+
+        .labels {
+            .label {
+                width: 100%;
+            }
+
+            .title {
+                padding-top: 0;
+                color: $verification-dropbox-title-color;
+            }
+
+            .subtitle {
+                color: $verification-dropbox-subtitle-color;
+            }
+        }
+    }
+}
+
+.verification-dropbox {
+    height: 100%;
+    background-color: mix($verification-blue, $white, 10%);
+
+    .dropbox {
+        height: 100%;
+
+        .content {
+            flex-direction: column;
+
+            img {
+                width: 160px;
+                height: auto;
+                margin: 0 auto;
+                padding: 12px 0 0 12px;        }
+
+            .labels {
+                text-align: center;
+
+                .label {
+                    width: 100%;
+                    font-size: 20px;
+                }
+
+                .title {
+                    font-size: 20px;
+                    font-weight: 600;
+                }
+
+                .subtitle {
+                    font-size: $font-size-medium;
+                    line-height: 1.5rem;
+                }
+            }
+        }
+    }
+}
+
+.verification-file-selector {
+    position: relative;
+    height: auto;
+    width: calc(100% - #{$verification-dropbox-border-width});
```

Comment:
width: 100%; should be fine with border-box. If the border is cropped at the right side, I'm pretty sure it's a subpixel rendering issue. If I set the border width to 1px it's cropped on chrome on my notebook monitor where I have a higher pixel ratio, if I move the window to my other monitor with a 1:1 pixel ratio, the border is shown.

Reply:
Fied it